R/dbetabinom.R

```diff
@@ -150,20 +156,34 @@ dbetaMix <- Vectorize(dbetaMix, vectorize.args = "x")
 #' @typed lower.tail : flag
 #'  if `TRUE` (default), probabilities are `P[X <= x]`,
 #'  and otherwise `P[X > x]`.
+#' @typed skipchecks : flag
```

Comment:
can you time how much this saves? I would think it is very little time difference, because `checkmate` is very fast.

Reply:
Yes, this change alone saves 2 minutes on building the vignette on my machine (~2x faster). While the checks themselves individually are fast, `pbetaMix` is called repeatedly in `qbetaMix` in `uniroot()`, also in `postprob`. So we end up checking the same arguments very many times when those arguments should be correct from the parent functions.

I propose now to have an internal non-exported function `.pbetaMix()` called from `pbetaMix()` and remove the `skipchecks`. Then `qbetaMix` can call `.pbetaMix` directly.

Reply:
Thanks that sounds good, good idea